src/r3_hex_core.erl

```diff
@@ -34,11 +34,11 @@
 %%    to the repository URL and overrides the `repo_name' option.
 %% * `repo_verify' - If `true' will verify the repository signature (default: `true').
 %% * `repo_verify_origin' - If `true' will verify the repository signature origin,
-%%   requires protobuf messages as of r3_hex_core v0.4.0 (default: `true').
+%%   requires protobuf messages as of hex_core v0.4.0 (default: `true').
 
 -module(r3_hex_core).
 
--include("r3_hex_core.hrl").
+-include("hex_core.hrl").
```

Comment:
oops, that should have been `r3_hex_core.hrl`, will fix the script.

Reply:
k, no worries, can just re-sync

Reply:
done!

Reply:
Looks good